htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -72,6 +72,8 @@ INSERT INTO llx_accounting_system (fk_country, pcg_version, label, active) VALUE
 ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime NULL;
+ALTER TABLE llx_product_lot ADD COLUMN commissionning_date datetime NULL;
+ALTER TABLE llx_product_lot ADD COLUMN qc_frequency int;
```

Comment:
Same here, for the moment this field must be first into llx_product.

Reply:
The problem is that you add it into llx_product.sql but not in migration files

Reply:
Yes they are, line 423